searches/binary_search.py

```diff
@@ -219,7 +228,7 @@ def binary_search(sorted_collection, item):
     return None
 
 
-def binary_search_std_lib(sorted_collection, item):
+def binary_search_std_lib(sorted_collection, item) -> Optional[int]:
```

Comment:
I think you missed a few in here :)

Reply:
Good catch! Looks like @cclauss has taken care of this